includes/modules/payment/paypal/paypalwpp_incontext.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * paypal EC In-Context button display template
+ *
+ * @package paymentMethod
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Wed Aug 17 03:03:03 2016 -0500 Modified in v1.6.0 $
+ */
+
+    require_once(DIR_WS_CLASSES . 'payment.php');
+    $paypalwpp_module = 'paypalwpp';
+    // init the payment object
+    $classPay = new payment($paypalwpp_module);
+?><script>
+window.paypalCheckoutReady = function () {
+  paypal.checkout.setup('<?php echo MODULE_PAYMENT_PAYPALWPP_MERCHANTID; ?>', {
+    //locale: '<?php echo ${$paypalwpp_module}->getLanguageCode('incontext'); ?>',
```

Comment:
So, here was trying to align various code to existing usage. I need to relocate from where this was taken and also verify that on first usage (shopping_cart) that the language identifier is returned. Reason for this verification is that when first implemented, received an odd error with a section of notifier code where the issues appeared unrelated. But once the require of the payment class was made require_once, the issue went away. It was then seen as odd that ${$paypalwpp_module} actually worked. So in a way this syntactical oddity was left to in a way make a point and possibly address a separate issue. If $classPay really represents the resulting class assignment, then it seems that the corrected code should be $classPay->.

To further "explain", I had found something that worked in the heat of solving the problem and hadn't gone back to perform further potential cleanup such as this. It just doesn't seem right that in calling to declare a class that suddenly the global space now includes the variable by that modules class. Not saying it's a problem, just seemed unusual and another reason why the code wasn't presented in a more "syntactical" fashion. :)